### PR TITLE
documentReference w basedOn - reference -> specimen fixture

### PR DESCRIPTION
**Pull request by teslajoy**

### CODE REVIEW

**Comment by bwalsh** [comment]
LGTM.  @quinnwai can you merge?

File: tests/fixtures/fhir-gdc-examples/META/Patient.ndjson [hunk 1]
```diff
@@ -0,0 +1 @@
..."},{"extension":[{"url":"ombCategory","valueCoding":{"system":"urn:oid:2.16.840.1.113883.6.238","code":"2106-3","display":"White"}},{"url":"https://ncit.nci.nih.gov","valueCoding":{"system":"https://ncit.nci.nih.gov/ncitbrowser/ConceptReport.jsp?dictionary=NCI_Thesaurus&ns=ncit&code=C41261","code":"C41261","display":"white"}}],"url":"http://hl7.org/fhir/us/core/StructureDefinition/us-core-race","valueString":"White"},{"extension":[{"url":"ombCategory","valueCoding":{"system":"urn:oid:2.16.840.1.113883.6.238","code":"2186-5","display":"Not Hispanic or Latino"}},{"url":"https://ncit.nci.nih.gov","valueCoding":{"system":"https://ncit.nci.nih.gov","code":"C41222","display":"not hispanic or latino"}}],"url":"http://hl7.org/fhir/us/core/StructureDefinition/us-core-ethnicity","valueString":"not hispanic or latino"},{"url":"http://hl7.org/fhir/SearchParameter/patient-extensions-Patient-age","valueQuantity":{"value":72}}],"identifier":[{"system":"https://gdc.cancer.gov/case_submitter_id","value":"TCGA-38-7271"},{"system":"https://gdc.cancer.gov/case_id","value":"8214a0d1-5e2d-4a7a-acb1-e5580755db83"}],"gender":"female","birthDate":"1934-01-01","deceasedDateTime":"1970-01-01T00:33:28+00:00"}
```
Comment (quinnwai) [inline tests/fixtures/fhir-gdc-examples/META/Patient.ndjson:1]: Noticed there's an extension nested in an extension, was this intentional?

**Review by quinnwai — COMMENTED** [review]
quick question on a comment

File: tests/fixtures/fhir-gdc-examples/META/Patient.ndjson (hunk 1 above)
Comment (bwalsh) [reply tests/fixtures/fhir-gdc-examples/META/Patient.ndjson:1]: Legal FHIR:

Any resource can have an extension:
image

An extension is an element, that in turn can have an extension:
image

Handled here:
https://github.com/ACED-IDP/gen3_util/blob/feature/stateful-emitter/tests/unit/dataframer/dataframer.py#L47-L48

**Pull request merged**